benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/internal_data/hammerdb_vm_template.yaml

```diff
@@ -74,14 +74,14 @@ spec:
       kind: vm
       client_vm:
         dedicatedcpuplacement: false
-        sockets: 1
-        cores: {{ cores }}
+        sockets: {{ vm_sockets }}
+        cores: {{ vm_cores }}
         threads: 1
         image: quay.io/cloud-bulldozer/{{ db_image }}
         limits:
-          memory: {{ limits_memory_GB }}Gi
+          memory: {{ vm_limits_memory_GB }}Gi
```

Comment:
I would recommend removing memory limits for VMs as it can cause OOM if qemu mem overhead is a bit too high. In general I don't think we need to set limits for VMs specifically because they are already bound by the guest size (unlike pods/kata).

Reply:
OK. good point I will remove it and test.